docs/tasks/upgrading/upgrading-0.8-0.9.rst

```diff
@@ -2,4 +2,17 @@
 Upgrading from v0.8 to v0.9
 ===========================
 
-There are no special notes or considerations when upgrading from v0.8 to v0.9.
+Due to a change in the API group that cert-manager deployments use
+(`apps/v1beta1` to `apps/v1`), cert-manager deployments must first be deleted
+before applying the new version. This will cause downtime until the new version
+has been applied. To perform this action run:
```

Comment:
Can you explicitly mention that no data will be lost as a result of this? Else people will start worrying!

Might *also* be worth advising people to read the Backup guide first too, just in case? (better safe than sorry..)

Reply:
:+1: